packages/common/test/pipes/keyvalue_pipe_spec.ts

```diff
@@ -84,6 +84,28 @@ describe('KeyValuePipe', () => {
       const pipe = new KeyValuePipe(defaultKeyValueDiffers);
       expect(pipe.transform(value)).toEqual(null);
     });
+
+    it('should accept an object with optional keys', () => {
+      interface MyInterface {
+        one?: string;
+        two?: number;
+        three?: string;
+        four?: string;
+      }
```

Comment:
Let's probably use TypeScript's utility Types  (`Partial<Interface>`) instead of manually set every property of the interface to optional with **'?'** .
```suggestion
      interface MyInterface {
        one: string;
        two: number;
        three: string;
        four: string;
      }
```